lib/include/binder_ext_call_impl.h

```diff
@@ -77,6 +77,8 @@ typedef struct binder_ext_call_interface {
         BinderExtCallDisconnectFunc handler, void* user_data);
     gulong (*add_ring_handler)(BinderExtCall* ext,
         BinderExtCallFunc handler, void* user_data);
+    gulong (*add_ringback_tone_handler)(BinderExtCall* ext,
+        BinderExtCallRingbackToneFunc handler, void* user_data);
     gulong (*add_ssn_handler)(BinderExtCall* ext,
```

Comment:
To maintain binary compatibility, new methods must be added after the existing ones, i.e. replace of the reserved slots.

Reply:
Fixed